examples/using-remark/src/pages/2016-04-15---hello-world-kitchen-sink/index.md

```diff
@@ -11,14 +11,14 @@ tags:
 ![](joshua-earle-234740.jpg)
 *Photo by [Joshua Earle](https://unsplash.com/@joshuaearle) via [Unsplash](https://unsplash.com/@joshuaearle?photo=-87JyMb9ZfU)*
 
-**_Note how all headlines below show an anchor link when you hover them?_**  
```

Comment:
;) Sorry :( … Looking at using-remark, I think I tend to overuse that feature by now –
maybe to make up for all the times that my editor settings ate them up when editing other peoples docs?

FWIW, I started using Atom last year and its whitespace package [takes care of that](https://github.com/atom/whitespace/pull/82) nicely.